### DISCUSSION TITLE
Let Miss no longer affect accuracy PP

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#26228] [Ideas] [answered]
Break combo without obtaining a miss? Losing the slider tick can achieve this without losing the acc pp. But why can't miss be implemented?

If I miss a circle, it will significantly reduce the acc pp. However, in reality, the reason why the player miss the circle is more likely due to aiming, and the player did not click on the circle. We don't know the player's offset this time, why deduct the acc pp?

%

I mean. The calculation method of acc pp can be changed to only calculate the hit circle, and only check the accuracy of the hit circle, ignoring miss.

%

If you are still worried about players abusing miss to obtain high acc pp...
('... 'refers to the entire formula originally used to calculate acc pp)
-It is also possible to add "(...) * (n)/(n+10)" (n=hit count) to require players to achieve higher hit counts, it also fixes the issue of easily obtaining higher acc pp on overly short beatmap.
-Additionally, the above can be changed to "(...) * (n ^ 2)/(n ^ 2+10 ^ 2)", which looks more extreme and solves the problem that even if a player hits 100 circles, it does not prove that 9.09% is the score they deserve. And further limiting the problem of achieving high scores even with too few hits on the circle. This is the most recommended because it has the best of both worlds.
-Or "(...) * (1-m/n)" (n=total number of circles, m=number of miss) to punish players with excessive miss.

%

However, players may intentionally miss some more difficult to perfectly hit circles to ensure the accuracy pp? Okay, don't worry too much, because that way the score will be very low, and the aim pp will be significantly reduced. At least doing so can help players prove how accurate they are, rather than completely following combo and aim.

**Comment by Rekunan** [comment, marked as answer]
If you look at the [implementation of acc pp](https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs#L187C7-L187C7), there is no miss penalty. Any reduction in acc pp from misses is due to misses affecting accuracy itself, and/or a miss causing a lack of 300/100/50.
